openff/interchange/_tests/unit_tests/smirnoff/test_charge_assignment_logging.py

```diff
@@ -149,7 +167,7 @@ def map_methods_to_atom_indices(caplog: pytest.LogCaptureFixture) -> dict[str, l
 
 
 @pytest.fixture
-def sage_with_nagl(sage):
+def sage_with_nagl_chargeincrements(sage):
```

Comment:
Note to reviewer: I changed this to disambiguate between NAGL called via ChargeIncrementModelHandler (where it tests some SMIRKS-based chargeincrement stuff as well) vs NAGLChargesHandler